packages/amplication-nest-logger-module/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "es2016" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
I meant to remove _all_ comments from `tsconfig.json` :) If we need to we can check out the docs. I don't think it looks very professional.